FightPandemics/FightPandemics-iOS
# Add Custom Font Files

**What this PR does**:
This PR adds custom font files to the project

**What issue this PR fixes**:
#29 

**Steps to test**:
Just for testing to see if the custom fonts works, add a label to a storyboard and change the font to custom fonts with the fonts added. Then see if applies as expected.
